src/index.html

```diff
@@ -1,14 +1,56 @@
 <!doctype html>
 <html lang="en">
   <head>
+
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,700;1,400&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a href="index.html" class="logo__img">
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" data-qa="hover"
+            href="#laptops & Computers">Laptops & Computers</a>
```

Comment:
```suggestion
            <a 
              class="nav__link" 
              data-qa="hover"
              href="#laptops & Computers"
            >
              Laptops & Computers
            </a>
```

Reply:
not fixed 
https://github.com/mate-academy/layout_moyo-header/pull/4101/files#r1390796143